docs/Reference/CLI/Subcommands/Admin.md

```diff
@@ -0,0 +1,550 @@
+---
+title: Subcommand options
+---
+
+# `admin`
+
+Used to perform administrative tasks. To use the `admin` subcommand run:
+
+```bash
+teku admin [subcommand] [subcommand] [options]
+```
+
+You can also supply the subcommand options using [environment variables or a configuration file].
+
+## `weak-subjectivity`
+
+Display or clear weak subjectivity configuration. 
+
+### `clear-state`
+
+Clears the stored weak subjectivity configuration.
+
+#### config-file
+
+=== "Syntax"
+
+    ```bash
+    teku admin weak-subjectivity clear-state --config-file=<FILE>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku admin weak-subjectivity clear-state --config-file=/home/me/me_node/config.yaml
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    TEKU_CONFIG_FILE=/home/me/me_node/config.yaml
+    ```
+
+The path to the YAML configuration file.
+The default is `none`.
+
+#### data-base-path, data-path
+
+=== "Syntax"
+
+    ```bash
+    teku admin weak-subjectivity clear-state --data-base-path=<PATH>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku admin weak-subjectivity clear-state --data-base-path=/home/me/me_node
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    TEKU_DATA_BASE_PATH=/home/me/me_node
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    data-base-path: "/home/me/me_node"
+    ```
+
+The path to the Teku data directory. The default directory is OS dependent:
+
+* macOS: `~/Library/teku`
+* Unix/Linux: `$XDG_DATA_HOME/teku` if `$XDG_DATA_HOME` is set; otherwise `~/.local/share/teku`
+* Windows: `%localappdata%\teku`.
+
+The default Docker image location is `/root/.local/share/teku`.
+
+#### data-beacon-path
+
+=== "Syntax"
+
+    ```bash
+    teku admin weak-subjectivity clear-state --data-beacon-path=<PATH>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku admin weak-subjectivity clear-state --data-beacon-path=/home/me/me_beacon
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    TEKU_DATA_BEACON_PATH=/home/me/me_beacon
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    data-beacon-path: "/home/me/me_beaon"
+    ```
+
+Path to the beacon chain client data. Defaults to `<data-base-path>/beacon` where `<data-base-path>`
+is specified using [`--data-base-path`](#data-base-path-data-path).
+
+#### data-storage-archive-frequency
```

Comment:
Some of these options aren't really required for these new `admin weak-subjectivity` commands.  I can look into cleaning up the help text.  But in the meantime, should we document all of these options?  The ones that are not really relevant are: `data-validator-path`, `data-storage-archive-frequency`, `data-storage-mode`, `initial-state`.

Reply:
As discussed, will remove these options once it's removed from the code.